### ISSUE TITLE
Import through share action from other apps

### ISSUE THREAD

**Issue by PinkPlasticBeach** [Ethereal-Developers-Inc/OpenScan#50]
Tried to share a jpg from gallery to crop and convert to pdf in openscan.
(Tried open gallery and stock gallrry of lineageos)

Openscan is not listed in the apps i can share images with.

This would enhance workflow a lot. Finding the folder where images are can be tiring as they may originate in many different applications (camera, chats/messengers, evernote, collaborative clouds....)

Thx for openscan!

**Comment by yephny** [comment]
Agreed, this should be top priority as it's easy to implement and necessary/useful.
